package.json

```diff
@@ -103,13 +103,13 @@
     "lint-staged": "15.2.2",
     "mocha-junit-reporter": "2.2.1",
     "prettier": "2.8.8",
-    "qlik-chart-modules": "0.67.0",
+    "qlik-chart-modules": "0.68.0",
     "shx": "0.3.4",
     "typescript": "5.3.3",
     "yargs": "17.7.2"
   },
   "peerDependencies": {
-    "@nebula.js/stardust": ">=4.11.0"
+    "@nebula.js/stardust": ">=4.12.0"
```

Comment:
I would say no and we should change it to `^4.x` unless some specific minor version is required. Then also disable renovate from updating peerDependencies.

The main issue with updating peer deps version is that it can be seen as a breaking change for the consumer of sn-action-button package.